src/main/java/org/jitsi/jicofo/bridge/Bridge.java

```diff
@@ -223,6 +243,12 @@ public void setStats(ColibriStatsExtension stats)
         {
             this.relayId = relayId;
         }
+
+        String healthy = stats.getValueAsString("healthy");
+        if (healthy != null)
+        {
+            this.healthy = Boolean.parseBoolean(healthy);
+        }
```

Comment:
We don't expect a null here, right? Maybe we could print a warning message, if this happens?